packages/engine/Source/Scene/Cesium3DTileset.js

```diff
@@ -2709,7 +2824,34 @@ function processTiles(tileset, frameState) {
       --statistics.numberOfTilesProcessing;
       handleTileFailure(error, tileset, tile);
     }
+
+    if (tileset.totalMemoryUsageInBytes > cacheByteLimit) {
+      memoryExceeded = true;
+      break;
+    }
+  }
+
+  if (tileset.totalMemoryUsageInBytes < cacheBytes) {
+    decreaseScreenSpaceError(tileset);
+  } else if (memoryExceeded && tiles.length > 0) {
+    increaseScreenSpaceError(tileset);
+  }
+}
+
+function increaseScreenSpaceError(tileset) {
+  tileset._memoryAdjustedScreenSpaceError *= 1.02;
+  const tiles = tileset._processingQueue;
+  for (let i = 0; i < tiles.length; ++i) {
+    tiles[i].updatePriority();
   }
+  tiles.sort(sortTilesByPriority);
```

Comment:
I wonder if we should always be updating priority and sorting, not just when the SSE changes. Because it doesn't seem strictly necessary to just do it here.

Also is there a reason it's updated here and not in `decreaseScreenSpaceError`?